app/models/comment.rb

```diff
@@ -232,52 +232,40 @@ def shorten_urls!
   end
 
   def calculate_score
-    update_column(:score, BlackBox.comment_quality_score(self))
-    update_column(:spaminess_rating, BlackBox.calculate_spaminess(self))
-    root.save unless is_root?
+    Comments::CalculateScoreJob.perform_later(id)
   end
-  handle_asynchronously :calculate_score
 
   def after_create_checks
     create_id_code
     touch_user
   end
 
   def create_id_code
-    update_column(:id_code, id.to_s(26))
+    Comments::CreateIdCodeJob.perform_later(id)
   end
-  handle_asynchronously :create_id_code
 
   def touch_user
-    user.touch(:updated_at, :last_comment_at)
+    Comments::TouchUserJob.perform_later(id)
   end
-  handle_asynchronously :touch_user
 
   def expire_root_fragment
     root.touch
   end
 
   def create_first_reaction
-    Reaction.create(user_id: user_id,
-                    reactable_id: id,
-                    reactable_type: "Comment",
-                    category: "like")
+    Comments::CreateFirstReactionJob.perform_later(id)
   end
-  handle_asynchronously :create_first_reaction
 
   def before_destroy_actions
     commentable.touch(:last_comment_at) if commentable.respond_to?(:last_comment_at)
     remove_notifications
-    bust_cache_without_delay
+    bust_cache
```

Comment:
`bust_cache_without_delay` wasn't run asynchronously, so to preserve logic you need to call busting cache synchronously.
If you keep the `_without_delay` methods as specified in the above comment, you can just keep the previous `bust_cache_without_delay` call.